src/go/configinfo/service_info.go

```diff
@@ -161,20 +168,20 @@ func NewServiceInfoFromServiceConfig(serviceConfig *confpb.Service, id string, o
 	return serviceInfo, nil
 }
 
-func (s *ServiceInfo) buildLocalBackend() error {
-
-	scheme, hostname, port, _, err := util.ParseURI(s.Options.BackendAddress)
+func (s *ServiceInfo) buildBackend(address string, isTestOnlyHTTPBackend bool) error {
```

Comment:
keep it as `buildLocalBackend`.    We have local and remote,   remote are from `Backend.address`,  local is from the flag.    Maybe `buildBackendFromFlag`?

Reply:
Combining your comment, I changed it to buildLocalBackendFromAddress.